pandas/io/formats/style.py

```diff
@@ -181,6 +181,8 @@ def __init__(
         self.cell_ids = cell_ids
         self.na_rep = na_rep
 
+        self.tooltips = _Tooltips()
```

Comment:
shouldn't this just be None by default? and if set_toolstips is called *then* it has a value)

Reply:
I have changed to your preference. Was that structure originally to avoid some mypy issues which I have worked around and since the initialised `_Tooltips()` instance was pretty empty anyway.